### DISCUSSION TITLE
Struggle with Istio and Vault as CA integration.

### DISCUSSION THREAD

**Discussion by k8stuff** [istio/istio#50167] [Q&A]
I am looking at the following istio blog: https://istio.io/v1.2/docs/tasks/security/vault-ca/

the helm chart is referencing a template within the install folder structure but the install files for istio doesn’t have the folder structure. 

What am I missing please ? Are there updated documents with a bit more information?

**Comment by howardjohn** [comment]
The doc you linked is about 5 years old; first class vault support was removed shortly after. So I wouldn't follow.things there. 

I'm not familiar enough with vault to suggest alternatives but from some quick googling seems like there are a few community integrations

**Comment by k8stuff** [comment]
I tried to look. Maybe my Google foo let me down. If you found a community effort I’d appreciate if you could post a link

**Comment by abasitt** [comment]
You may look in to cert-manager + istio-csr + vault combination.

**Comment by rootsongjc** [comment]
@k8stuff Maybe this blog can help you, check it out https://tetrate.io/blog/how-to-use-hashicorp-vault-as-a-more-secure-way-to-store-istio-certificates/
